js/index.js

```diff
@@ -1,6 +1,93 @@
 // Iteration 1: Names and Input
+//1.1
+let hacker1 = "Emanuel"
+//1.2
+console.log(`The driver's name is ${hacker1}`)
+//1.3
+let hacker2 = "Roman"
+//1.4
+console.log(`The navigator's name is ${hacker2}`)
 
-// Iteration 2: Conditionals
+//Iteration 2: Conditionals
+//2.1
+let driver = hacker1.length
+let navigator = hacker2.length
 
+if(driver>navigator){
+    console.log(`The driver has the longest name, it has ${driver} characters.`)
 
-// Iteration 3: Loops
+}else if(navigator>driver){
+    console.log(`The navigator has the longest name, it has ${navigator} characters.`)
+}else if (navigator===driver) {
+    console.log(`Wow, you both have equally long names, ${navigator} characters!`)
+}
+//Iteration 3: Loops
+//3.1
+let prueba = String()
+for(let i = 0; i < driver; i++){
+  let array = " "
+  let word = hacker1[i].toUpperCase() + array
+  prueba = prueba + word
+
+}
+console.log(prueba)
+//3.2
+let pruebaDos = String()
```

Comment:
En esta iteracion el loop deberia empezar en el ultimo elemento de nuestro string e ir iterando de atras hacia adelante para ir concatenando cada letra 👀 
```suggestion
let pruebaDos = "";
for(let i = hacker2.length - 1; i >= 0; i--) {
  pruebaDos += hacker2[i];
}
console.log(pruebaDos);```